tutorials/easydeploy-grafana/index.mdx

```diff
@@ -0,0 +1,67 @@
+---
+meta:
+  title: Deploy ArgoCD on Scaleway Kubernetes clusters using Easy Deploy
```

Comment:
Metadata are wrong, this page is about Grafana

Reply:
```suggestion
  title: Deploy Grafana on Scaleway Kubernetes clusters using Easy Deploy
```